WakaTime/WakaTime.swift

```diff
@@ -18,7 +18,11 @@ struct WakaTime: App {
 
     init() {
         registerAsLoginItem()
-        downloadCLI()
+        Task {
+            if !(await Self.isCLILatest()) {
```

Comment:
Note: This now runs in its own thread. So I have made the functions static to avoid concurrent writes to mutable state.